docs/user/03-traces.md

```diff
@@ -103,16 +111,16 @@ spec:
     randomSamplingPercentage: 5.00
 ```
 
-### Step 3a: Add Authentication Details From Plain Text
+### 3a: Add Authentication Details From Plain Text
 
-To integrate with external systems, you must configure authentication details. At the moment, mutual TLS (mTLS), Basic Authentication and custom headers are supported.
+To integrate with external systems, you must configure authentication  details. You can use mutual TLS (mTLS), Basic Authentication, or custom headers:
 
 <!-- tabs:start -->
 
 #### **Mutual TLS**
 
 ```yaml
-apiVersion: telemetry.kyma-project.io/v1alpha1
+apiVersion: telemetry.kyma-project.io/v1
```

Comment:
That is wrong, only the istio resource version changed!
```suggestion
apiVersion: telemetry.kyma-project.io/v1alpha1
```

Reply:
🙈